keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -478,46 +487,84 @@ Because Windows privileged containers will work much differently than Linux priv
-  - Note: it is not possible to feature-gate this behavior in client libraries and because of this the functionality should not be added to client libraries after `hostProcess` containers while this feature is in `alpha`.
-  - [kubernetes/kubernetes#104490](https://github.com/kubernetes/kubernetes/pull/104490) adds support for `HostProcess` containers to the golang client library.
-- Named Pipe mounts will **not** be supported. Instead named pipes should be accessed via their path on the host (\\\\.\\pipe\\*).
-  - The following error will be returned if `hostProcess` containers attempt to use name pipe mounts - https://github.com/microsoft/hcsshim/blob/358f05d43d423310a265d006700ee81eb85725ed/internal/jobcontainers/mounts.go#L40.
-- Unix domain sockets mounts support will be added before `HostProcess` containers graduate to `stable`. For `alpha` and `beta` Unix domain sockets can be accessed via their paths on the host like named pipes.
-  - The Windows APIs needed to support mounting unix domain socket mounts in `hostProcess` containers was introduced in Windows Server Ver, 2004. Microsoft is planning on backporting these APIs to Windows Server 2019 (min support Windows Server OS) to provide a consistent user experience.
-- Mounting directories from the host OS into `hostProcess` containers will work just like with normal containers - kubelet will explicitly block this scenario.
-- All other volume types supported for normal containers on Windows will work with `hostProcess` containers.
+There will be two different behaviors for how volume mounts are configured in `hostProcess` containers.
+
+- **Bind Mounts**
+  - With the approach Window's bind-filter driver will be used to create a view that merges the host's OS filesystem with container-local volumes.
+  - When `hostProcess` containers are started a new volume will be created which contains the contents of the contaner image.
+    This volume will be mounted to `c:\hpc`.
+  - Additional volume mounts specified for `hostProcess` containers will be mounted at their requested location and can be access the same way as volume mounts in Linx or regular Windows Server contaienrs.
+    - ex: a volume with a mountPath of `/var/run/secrets/token` for containers will be mounted at `c:\var\run\secrets\token` for containers.
+  - Volume mounts will only be visible to the containers they are mounted into.
+  - The default working directory for `hostProcess` containers will also be set to `c:\hpc`.
+  - If a volume is mounted over a path that already exists on the host then the contents of the directory of the host, only the contents of the mounted volume will be visiable to the `hostProcess` container. This is the same behavior as regular Windows server behaviors.
+    - A `warn` message will be written to the containerd logs if a volume is being mounted at a location that already exists on the host.
+
+- **Symlinks**
```

Comment:
are there any new escalation paths or issues with symlink-following that would allow unintented access to or modification of host files / directories?

Reply:
The symlink volume mount behavior is the current alpha/beta behavior.
We aren't making any changes to this behavior (except for possibly more user-facing documentation) so there shouldn't be any new escalation paths or anything.

Reply:
There shouldn't be.
Using `HostPath` volume mounts in non-hostprocess containers in Windows containers would give the same access to file-system resources (largely due to how user accounts from inside containers get mapped to host filesystem access).

We can make sure this is documented appropriately.
W.R.T. pod security standards both sets of functionality are forbidden in baseline/restricted profiles.